autoPyTorch/api/tabular_classification.py

```diff
@@ -453,6 +442,43 @@ def search(
             portfolio_selection=portfolio_selection,
         )
 
+    def _get_dataset_compression_mapping(
```

Comment:
Actually we do not use self, so we can define it as a function and share it with other classes.

Reply:
True, that's better I think